internal/keepers/filter_test.go

```diff
@@ -255,5 +256,5 @@ func TestReportCoordinator(t *testing.T) {
 
 func assertFilter(t *testing.T, reg *types.MockRegistry, key types.UpkeepKey, id types.UpkeepIdentifier, exp bool, f func(types.UpkeepKey) bool) {
 	reg.Mock.On("IdentifierFromKey", key).Return(id, nil)
-	assert.Equal(t, exp, f(key), "filter should return '%v' to indicate key should not be filtered out at block %s", exp, string(key))
+	assert.Equal(t, exp, f(key), "filter should return '%v' to indicate key should not be filtered out at block %s", exp, key.String())
```

Comment:
You don't need to do .String(), it would be done automatically if the type implements stringer interface